.github/workflows/test.yml

```diff
@@ -5,14 +5,47 @@ on: [push, pull_request]
 jobs:
   test:
     runs-on: ubuntu-latest
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - 5000:5000
     steps:
-    - uses: actions/checkout@v2
-    - run: npm ci
-    - run: npm audit --production
-    - run: npm test -- --testPathIgnorePatterns action.test.js
+      - uses: actions/checkout@v2
+      - run: echo $(uname -a)
+      - name: Check for npm (so make test works)
+        run: |
```

Comment:
Do you think it's worth trying to have all of these steps be available as make tasks? (For local use)

Reply:
these are not required for local use, running `make test` will cause this workflow to be executed and that's why it's needed because `act` is giving a container without node for some unexplained reason. but ALSO it's just easier to install `node` & `docker` & add the registry container and then run `jest` directly because you can easily debug the code and such.